components/o-forms/src/scss/_toggle.scss

```
@@ -60,6 +60,16 @@
 			&:invalid:not(:disabled) + .o-forms-input__label:after { // stylelint-disable-line selector-no-qualifying-type
 				border-color: inherit;
 			}
+			&:focus-visible {
+				outline: none;
+				box-shadow: none;
+			}
+
+			&:focus + .o-forms-input__label::before {
+				@if oBrandIs('core') or oBrandIs('internal') {
+					outline: 4px solid oColorsByUsecase(focus, 'outline'); // oCoulorsByUsecase doesn't support whitelable brand
+				}
```

We can pass `null` to the 3rd [fallback parameter](https://registry.origami.ft.com/components/o-colors@6.4.2/sassdoc?brand=core#function-ocolorsbyusecase) to handle this.

```suggestion
				outline: 4px solid oColorsByUsecase('focus', 'outline', $fallback: null);
```

For the whitelabel brand that will output this, and the colour will be inherited:

```
				outline: 4px solid;
```